### PR TITLE
Use statistics model from oira.statistics.deployment

### PR DESCRIPTION
**Pull request by reinhardt**
Requires https://github.com/EU-OSHA/oira.statistics.deployment/pull/11

Ref syslabcom/scrum#2348

### CODE REVIEW

File: src/osha/oira/upgrade/alembic_statistics/env.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 from alembic import context
-from osha.oira.statistics.model import Base
+from oira.statistics.deployment.model import Base
```
Comment (ale-rt) [inline src/osha/oira/upgrade/alembic_statistics/env.py:2]: It surprises me to see the alembic upgrades for the `oira.statistic.deployment` package in this package.

**Review by ale-rt — COMMENTED** [review]
I am not really fond of pulling in the whole `oira.statistics.deployment` package.

It seems to me that you need this to take some data to one db, do something and store the result in another place.

I do not know if I am correct.

If I am right, you could even pip install both packages in a venv, setup the engines and you would have a working script.

Anyway it seems not a big deal because the dependencies of `oira.statistics.deployment` do not seem to be that much.

I would just reconsider the fact that the alembic upgrades have been kept in this module.

File: src/osha/oira/upgrade/alembic_statistics/env.py (hunk 1 above)
Comment (ale-rt) [reply src/osha/oira/upgrade/alembic_statistics/env.py:2]: > It shouldn't surprise you if you've read [syslabcom/scrum#2348](https://github.com/syslabcom/scrum/issues/2348). 

I read it once again and me still surprised :smile:.
I will setup a call to clarify this.

> The point is to correct the previous approach of doing alembic upgrades here in osha.oira and start doing them in oira.statistics.deployment instead. 

This is exactly why I am thinking that this file should not be here anymore.

> Maybe a good further step would be to remove the old upgrades that operate on the statistics databases from osha.oira.

Sure, I would not bother about that.

**Comment by reinhardt** [comment]
> If I am right, you could even pip install both packages in a venv

I'm reluctant to do that because osha.oira's dependencies pulls in a whole Plone installation.

**Review by ale-rt — APPROVED** [review]

**Comment by reinhardt** [comment]
The tests were failing because oira.statistics.deployment was not found. I decided to make it an optional dependency. That also should be handy for any dev or test installations that don't want to bother with the statistics.

**Review by ale-rt — APPROVED** [review]

**Review by ale-rt — APPROVED** [review]

**Pull request merged**